crates/bevy_render/src/camera/camera.rs

```diff
@@ -287,9 +287,15 @@ impl RenderTarget {
                     scale_factor: window.scale_factor(),
                 }
             }
-            RenderTarget::Image(image_handle) => {
-                let image = images.get(image_handle)?;
+            RenderTarget::Image(image_handles) => {
+                
+                let image = images.get(image_handles.first()?)?;
                 let Extent3d { width, height, .. } = image.texture_descriptor.size;
+                // Make sure all images are the same width/height.
+                assert!(image_handles.iter().all(|image_handle| {
```

Comment:
Instead of asserting we might want to error more gracefully here? I'm not super familiar with error stuff in bevy.